ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/_cq_dialog/.content.xml

```diff
@@ -34,6 +34,56 @@
                             jcr:primaryType="nt:unstructured"
                             sling:resourceType="granite/ui/components/coral/foundation/include"
                             path="core/fd/components/form/panelcontainer/v1/panelcontainer/cq:dialog/content/items/tabs/items/basic"/>
+                    <repeat
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Repeat Tabs on top"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                            margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <columns
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                    margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <repeatableText
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/text"
+                                                    text="Use the make tabs on top repeatable option to allow form users to dynamically add multiple panels to the form."/>
+                                            <repeatable
+                                                    jcr:primaryType="nt:unstructured"
+                                                    wrapperClass="cmp-adaptiveform-panelcontainer__repeatable"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/switch"
+                                                    name="./repeatable"
+                                                    fieldLabel="Make tabs on top repeatable"
```

Comment:
its business friendly name is `Horizontal Tabs`

Reply:
It needs to align with component name, which right now is 'Tabs on Top'